tools/gfx_tools/compose.py

```diff
@@ -517,23 +522,24 @@ def write_composite_png(self) -> bool:
         self.tileset.pngnum += self.sheet_width - \
             ((len(self.sprites) % self.sheet_width) or self.sheet_width)
 
-        if self.sprites:
```

Comment:
There is a check for empty self.sprites above already so this one was redundant.